### PR TITLE
very small typo and wording fixes for CAN doc

### PR DESCRIPTION
**Pull request by brainuser5705**

### CODE REVIEW

File: docs/source/canopen.rst [hunk 1]
```diff
@@ -41,13 +41,12 @@ can then be exposed on the CANopen network.
...
 would be a series of identifiers required by CANopen (specifically a vendor ID,
-product code, revision number, and serial number). Potentially a few control
-loop setting such as "max pump speed", "minimum threshold temperature", etc.
-Additionally the Object Dictionary may have data which is in the Object
-Dictionary which would allow other entities on the CANopen network to gain
-information on the state of the Temperature Management System such as
-"current pump speed", "current motor controller temperature", etc. An example
-CANopen Object Dictionary is shown in the sample below.
+product code, revision number, and serial number), potentially a few control
+loop setting such as "max pump speed", "minimum threshold temperature" or data
```
Comment (mjmagee991) [inline docs/source/canopen.rst:45]:
```suggestion
loop settings such as "max pump speed", "minimum threshold temperature" or data
```

File: docs/source/canopen.rst [hunk 2]
```diff
@@ -41,13 +41,12 @@ can then be exposed on the CANopen network.
 
 For example, take a Temperature Management System (TMS). In its Object Dictionary
```
Comment (mjmagee991) [inline docs/source/canopen.rst:42]: Can you break this sentence up? It's like 8 lines long and kind of hard to read

**Review by mjmagee991 — APPROVED** [review]
2 quick comments, but looks good

**Review by DiegoLHendrix — COMMENTED** [review]
looks good

**Review by Ethan-Caracoglia — APPROVED** [review]
The spelling corrections look good!

File: docs/source/canopen.rst [hunk 3]
```diff
@@ -40,13 +40,12 @@ from CANopen required identifiers, to device specific settings, to data which
...
-for other entities on the CANopen network to gain information on the state of 
-the Temperature Management System such as "current pump speed", "current motor 
-controller temperature", etc. An example CANopen Object Dictionary is shown 
-in the sample below.
+would first be the series of identifiers required by CANopen (specifically a 
+vendor ID, product code, revision number, and serial number). Then follows device
+settings which could be a few control loop settings such as "max pump speed" or 
+"minimum threshold temperature". The data to expose could be "current pump speed",
+"current motor controller temperature", etc. An example CANopen Object Dictionary
+is shown in :ref:`the sample<Looking at an Example>` below.
```
Comment (brainuser5705) [inline docs/source/canopen.rst:48]: added link to section heading

**Review by mjmagee991 — APPROVED** [review]
Nice work!

**Comment by mjmagee991** [comment]
Merging this because there are actually 2 approvals, but one person's account isn't set up correctly

**Pull request merged**